### DISCUSSION TITLE
Take Returns Wrong Number of Records?

### DISCUSSION THREAD

**Discussion by darrylmorley** [prisma/prisma#9246] [Q&A] [answered]
Hi,

I'm trying to implement pagination but `take` keeps taking a strange amount of records.

My query: 
```
const getItems = await prisma.product.findMany({
    skip: 0,
    take: 30,
    where: {
      category: {
        path: "fullPathName",
        string_starts_with: "SPARES & ACCESSORIES",
      },
    },
    orderBy: {
      description: "asc",
    },
  });
  ```
  
Currently `take: 10` returns 2 records, `take: 20` returns 8 and `take: 30` returns 10 records..

I'm using the postgresql provider.

**Comment by darrylmorley** [comment, marked as answer]
Rebuilt, working fine.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
